### PR TITLE
Add some Gradle tasks

### PR DESCRIPTION
**Pull request by Goldorion**
In my plugins, I started to use Gradle to simplify the repetitive tasks of building, installing and launching MCreator. So I decided to add them to this plugin template, so more users can use them to be quicker at testing their plugins. Current tasks are:

- Build the plugin
- Install it
- Launch MCreator

I also move files and folders related to the demo plugin inside an `src` folder, as the build task only works this way. I also renamed `1.14.4` and `1.15.2` folders to `forge-1.14.4` and `forge-1.15.2`.

### CODE REVIEW

**Comment by Goldorion** [comment]
How can I not push files into the `.idea` folder?

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (KlemenDEV) [inline .idea/workspace.xml]: This file should be in gitignore

Comment (Goldorion) [reply .idea/workspace.xml]: The file os already in the gitignore. This is why I asked how can I not push those files.

Comment (KlemenDEV) [reply .idea/workspace.xml]: When you used git add ., you should not add these. You will need to rm it probably

**Comment by Goldorion** [comment]
I found back the [code of BoogieMonster101](https://github.com/Goldorion/Fabric-Generator-MCreator/commit/66e1ea29d4b11e8aaeb55d1da093b06adf7008b3#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R40) for the launch task on mac and Linux, so I added it. I obviously can't test it, but as he didn't change it after, it should work.

**Review by KlemenDEV — CHANGES_REQUESTED** [review]
I would update Exporting readme section to include gradle task, rename build task to "export" and make new task group called "mcreator_plugin" for these tasks. Other than that, all good :)

**Review by KlemenDEV — APPROVED** [review]

**Pull request merged**